google_chat_ros/CardHeader header # The header of the card
google_chat_ros/Section[] sections # Sections are separated by a line divider
google_chat_ros/CardAction[] card_actions # The actions of this card
string name # Name of the card
